### PR TITLE
fix unwanted alert list marker for single error, issue #3583  

### LINKED ISSUE #3583: Minor Visual bug in Claiming a Pull payment, weird bullet point by providing invalid address
Yo, 

When you provide an invalid BTC address by claiming Funds from a Pull Payment, it gives a weird bullet point before the tekst. 

PullPayment-Bulletpoint

### PR DESCRIPTION
**Pull request by MrPaz**
should fix #3583 

noob who hasn't coded in a while getting back into coding.  please let me know if there is any better way to do this, or if I made any mistakes in the github / pull request process.  thank you!

Screen Shot 2022-05-02 at 2 42 18 PM

### CODE REVIEW

**Comment by dstrukt** [comment]
Visually, and logically (if only 1, hide <li> dot) looks good to me.

**Review by dstrukt — APPROVED** [review]

**Comment by dennisreimann** [comment]
Haven't tested it yet, but this looks like quite a large changeset for such a small thing.

**Comment by MrPaz** [comment]
yeah idk why the diff shows so many changes.  this is the only relevant change to the cshtml file:

Screen Shot 2022-05-03 at 2 15 28 PM

**Review by dennisreimann — CHANGES_REQUESTED** [review]
@MrPaz Can you bring in the changes I made on [my fork](https://github.com/dennisreimann/btcpayserver/tree/PullPaymentAlertBug) or give me access to your fork, so that I can push them to this PR branch?

I've fixed the indentation, simplified the condition and inlined the no-marker styles.

**Comment by NicolasDorier** [comment]
I did it, will merge when test pass

File: BTCPayServer/Views/UIPullPayment/ViewPullPayment.cshtml [hunk 1]
```diff
@@ -34,6 +34,15 @@
...
     @Safe.Raw(Model.EmbeddedCSS)
+    <style>
+        .no-marker > ul { list-style-type: none; }
+    </style>
+    <noscript>
+        <style>
+            .hide-when-js, [v-cloak] { display: block !important; }
+            .only-for-js { display: none !important; }
+        </style>
+    </noscript>
```
Comment (dennisreimann) [inline BTCPayServer/Views/UIPullPayment/ViewPullPayment.cshtml:45]: @NicolasDorier This part can be removed now that the PR is rebased. It is part of the `LayoutHead` partial included bove.

**Comment by dennisreimann** [comment]
@MrPaz Yes, I think it males sense to inline the styles in this case.

**Pull request merged**